config/routes.rb

```diff
@@ -300,7 +300,7 @@
         end
       end
 
-      resources :questions do
+      resources :questions, shallow: true do
         resources :answers, except: [:index, :destroy, :delete], controller: 'questions/answers', shallow: true do
```

Comment:
I'm pretty sure we don't need `shallow: true` here if we have it specified on `resources :questions`

Reply:
👍